onnxruntime/core/graph/graph.cc

```diff
@@ -301,13 +301,14 @@ void Node::SetNodeType(Node::Type node_type) noexcept {
   node_type_ = node_type;
 }
 
-const Function* Node::GetFunctionBody() const noexcept {
+Function* Node::GetFunctionBody() noexcept {
   return func_body_;
 }
 
-void Node::SetFunctionBody(const Function& func) {
-  func_body_ = &func;
-  op_ = &func.OpSchema();
+void Node::SetFunctionBody(Function* func) {
+  ORT_ENFORCE(nullptr != func);
```

Comment:
>ORT_ENFORCE(nullptr != func); [](start = 2, length = 29)

This isn't necessary. Use Function& and the API advertises clearly that nullptr isn't accepted